packages/cozy-procedures/src/redux/procedureDataSlice.js

```diff
@@ -0,0 +1,42 @@
+import get from 'lodash/get'
+import { createSlice } from 'redux-starter-kit'
+
+const procedureDataSlice = createSlice({
+  initialState: {
+    data: {
+      amount: null,
+      duration: null
+    },
+    ui: {
+      initiated: false
+    }
+  },
+  slice: 'procedureData',
+  reducers: {
+    updateAmount: (state, action) => {
+      state.data.amount = action.payload
+    },
+    updateDuration: (state, action) => {
+      state.data.duration = action.payload
+    },
+    updateStatus: (state, action) => {
+      state.ui.initiated = action.payload.initiated
```

Comment:
As far as I can see, we only ever changed it from the initial state to `true`. What do you think of changing the action to something like `initiationSuccess` without a payload?